tests/run_tests.sh

```diff
@@ -9,3 +9,5 @@ python -m pytest --nbmake $(echo $INCLUDED_NBS)
 
 INCLUDED_NBS=$(find ../nbs_tests -name "*.ipynb")
 python -m pytest --nbmake $(echo $INCLUDED_NBS)
+
+python -m pytest test_smiles.py
```

Comment:
I think `python -m pytest` will suffice, it discovers files that contain tests itself..

Reply:
thank you, fixed